buildscripts/winbuild.sh

```diff
@@ -134,27 +134,27 @@ function build_dll(){
 	cd src/bitmsghash || exit 1
 	if [ "${MACHINE_TYPE}" == 'x86_64' ]; then
 		echo "Create dll"
-		x86_64-w64-mingw32-g++ -D_WIN32 -Wall -O3 -march=native \
+		x86_64-w64-mingw32-g++ -D_WIN32 -Wall -O3 -march=x86-64 \
```

Comment:
why not just remove?

Reply:
Is that not what the  minus signs indicate?

I sure didn't intend to leave "march=native" so hopefully github is not doing that.

Reply:
I mean, aren't those values the defaults, that will be applied if you just remove the `-march=native`? Though maybe it's better to set that explicitly.

Reply:
It turns out that there is no default value for the march option.

> There is no default in general, because it's platform-specific.

https://gcc-help.gcc.gnu.narkive.com/Z259fPcD/default-value-for-march-option

Though you can find what your platform will use as default (if it is not specified) with the command:

gcc -Q --help=target 

https://stackoverflow.com/questions/27786932/what-is-the-default-for-gcc-march-option